docs/networking.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Networking
+subtitle: Connecting ev3dev to the internet and other devices
+---
+
+If you want to access your ev3dev installation from a PC or install new software
+on your EV3, you'll need to connect to a network. Here are some resources to
+help you out.
+
+# Connecting to the internet
+
+You have a few options for connecting to the internet.
+
+### With a Wi-Fi dongle
+
+The simplest route is to purchase a Wi-Fi dongle and plug it into the EV3's USB
+port. Almost any USB Wi-Fi dongle which supports Linux will work, including the
```

Comment:
Can you point me to a list of supported ones? I know of the Netgear one, but I was unaware that there were others that they supported. And are there any other dongles that you think are common and known to work that I should include?